openpype/hosts/maya/plugins/publish/extract_thumbnail.py

```diff
@@ -27,22 +29,46 @@ def process(self, instance):
 
         camera = instance.data["review_camera"]
 
-        maya_setting = instance.context.data["project_settings"]["maya"]
-        plugin_setting = maya_setting["publish"]["ExtractPlayblast"]
-        capture_preset = plugin_setting["capture_preset"]
+        host_name = instance.context.data["hostName"]
+        family = instance.data["family"]
+        task_data = instance.data["anatomyData"].get("task", {})
+        task_name = task_data.get("name")
+        task_type = task_data.get("type")
+        subset = instance.data["subset"]
+
+        filtering_criteria = {
+            "hosts": host_name,
+            "families": family,
+            "task_names": task_name,
+            "task_types": task_type,
+            "subset": subset
+        }
+
+        if self.profiles:
+            profile = filter_profiles(
+                self.profiles, filtering_criteria, logger=self.log
+            )
+            capture_preset = profile.get("capture_preset")
```

Comment:
Does this `ExtractThumbnail` ever get the `self.profiles` value if the settings are on the `ExtractPlayblast` class.

This seems invalid/broken.

Reply:
Good spot! Fixing